clone+annex

```diff
@@ -1,2 +1,5 @@
 FROM ubuntu
-RUN apt-get update
\ No newline at end of file
+RUN apt-get update && apt-get install python3-pip openssh-server openssh-client -y
+RUN apt-get install git git-annex -y
+RUN pip3 install snakemake
+
```

Comment:
What is the purpose of this file?

Reply:
This image is used in `Drone.yml` configs for pipeline steps.